deps/histogram/histogram.gyp

```diff
@@ -2,6 +2,7 @@
   'variables': {
     'histogram_sources': [
       'src/hdr_histogram.c',
```

Comment:
Why do we now pull in all files in `src` and `include` if they are not referenced here?

Reply:
I guess we dont need to, but  as for other dependencies, for a simpler update process